### DISCUSSION TITLE
Problem with mvn and app-authz-uma-photoz

### DISCUSSION THREAD

**Discussion by iriartmaxim** [keycloak/keycloak#10237] [Q&A] [answered]
![imagen](https://user-images.githubusercontent.com/97992995/154164362-36b1ea89-4e06-4846-8720-7697f074c4d8.png)

I am trying to do this but I am getting this error.
What could it be?

https://github.com/keycloak/keycloak-quickstarts/tree/latest/app-authz-uma-photoz

**Comment by pedroigor** [comment, marked as answer]
Could you try again, please? Some days ago, the jboss repo was facing issues.

**Reply by iriartmaxim** [reply]
Yes, i can now. Thanks you.
